pages/reference/base-images/base-images.md

```diff
@@ -108,92 +107,97 @@ CMD ["node", "main.js"]
 
 ### Supported Architectures, Distros and Languages
 
-Currently balenalib supports the following OS distributions and Language stacks, if you would like to see others added, create an issue on the [balena base images repo](https://github.com/balena-io-library/base-images/issues).
+Currently, balenalib supports the following OS distributions and Language stacks, if you would like to see others added, create an issue on the [balena base images repo]({{ $links.githubLibrary }}/base-images/issues).
+
+| Distribution | Default                      | Supported Architectures                      |
+| ------------ | ---------------------------- | -------------------------------------------- |
+| Debian       | Debian GNU/Linux 10 (buster) | armv5e, armv6, armv7hf, aarch64, amd64, i386 |
+| Alpine       | Alpine Linux v3.10           | armv6, armv7hf, aarch64, amd64, i386         |
```

Comment:
Hmmm, I've just noticed that this table says that the Default version of Alpine is v3.10, and by comparing Digests on https://hub.docker.com/r/balenalib/raspberrypi3-alpine/tags you can also confirm that the `latest` and `3.10` tags point to the same image, and yet Alpine 3.11 is mentioned in the "Multiple Distributions:" list on this page, and there's a `3.11` tag on DockerHub? :confused:

Reply:
_Maybe_ that's explained by https://github.com/balena-io/contracts/commit/84035e0310ef5f14fe3fd956986091084e5ca680#r36690619 but that's a bit of a random stab in the dark :wink:

Reply:
Yeah looks like a mistake as mentioned in that linked contract above. In general though `latest` doesn't necessarily mean "latest" 😕https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375  In this case though it looks like it should have been tagged latest.

Reply:
That's interesting, but I've no idea how and/or if Docker tags relate to the base-image tags used by Balena :wink:

Reply:
I'm not sure I understand that comment. The tags on Docker Hub are the base-image tags.

Reply:
The medium.com article you've linked to mentions tags like `marc/test:latest` and it seems like the `latest` tag there is automatically added by Docker. I was assuming that that's actually unrelated to the `latest` tag embedded within a Balena image-name like `balenalib/raspberrypi3-alpine-python:latest-3.7` ? :man_shrugging: :man_shrugging: